### DISCUSSION TITLE
"Intro to GitHub Flow" won't let me mark it as complete, even tho i completed it.

### DISCUSSION THREAD

**Discussion by TxmmyRL** [community/community#119665] [GitHub Education]
### Hi everyone,

the Introduction to Github called "Intro to GitHub Flow" won't let me mark it as complete. Is there a way to fix that bug or did I just miss a point and it is really not completed? The ReadMe of the session says:

Finish
Congratulations, you've completed this course and joined the world of developers!

, so i thought i would be finished.

Thanks for your help! :)

**Comment by KenchamVinay** [comment]
**hi @TxmmyRL ,** 

Let me know if you have resolved the issue, since i had the same problem with different repo. Although i completed the assignment correctly it shows that my assignment was Rejected.

**Reply by TxmmyRL** [reply]
No i did not resolve the issue yet. But found a way to continue anyways with the next repo somehow.

**Comment by KenchamVinay** [comment]
Could you guide me on how to do that? I am currently stuck on Communicate Using Markdown Assignment.

**Reply by lilimu996** [reply]
I had the same problem, you click on Explore more offers, there seems to be all his courses!
![image](https://github.com/community/community/assets/53130598/19d82207-bfd9-4d09-a89f-54411ec88518)

**Reply by KenchamVinay** [reply]
Got it. Thanks @lilimu996  , i was stuck on this for a day, i thought it was mandatary to complete this, since i choose a learning path at the beginning of the course.

**Comment by pabaz** [comment]
Hi, I am stuck at step1. After creating branch from the main, I have refreshed my page several times but it does not take me to step2 as stated.
Can anyone help me out.
Thank you

**Reply by TxmmyRL** [reply]
Hey, i had that issue too. I just created a whole new repo and then a new branch called "my-first-branch" in that new repo. That is what worked for me. I hope that helps.

**Reply by pabaz** [reply]
thanks, I figured it out. The .github folder has the steps folder in it so I used that

**Comment by jialianghua** [comment]
Hi, I have the same issue as you. Still hasn't resolved

**Reply by pabaz** [reply]
I hope you resolved it, if not creating a new project is the best way forward

**Comment by RkBeau** [comment]
Did anyone ever figure out how to get it to accept after a rejection.  I'm having the same issue on Explore Github.

**Reply by pabaz** [reply]
The solution for now is creating a new project

**Reply by Brandon-Isaac** [reply]
creating a new project seems not to be working

**Comment by 0motam** [comment]
I am having the same difficulty, I think it is best to skip for now

**Comment by potaxo** [comment]
I have the same problem. But I found this url : https://skills.github.com/, I hope this can help you.
